py/mpconfig.h

```diff
@@ -1581,6 +1581,12 @@ typedef double mp_float_t;
 #define MICROPY_PORT_BUILTINS
 #endif
 
+// Additional builtin function definitions for extension by commandine or variants.
+// See modbuiltins.c:mp_module_builtins_globals_table for format.
+#ifndef MICROPY_PORT_EXTRA_BUILTINS
+#define MICROPY_PORT_EXTRA_BUILTINS
```

Comment:
Would this naming scheme extend well to the other macros in this section, eg `MICROPY_PORT_EXTRA_ROOT_POINTERS`?  I guess it does (the alternative would be to append `_EXTRA`, or a similar word, to the name).

Reply:
I grepped for EXTRA already :)
Apart from in Makefiles it seems common to put it in the variable name instead of at the end.
I guess that is alphabetically sortable vs the variable naming convention generally used (adjectives before noun); I like the latter more because it feels more grammatically correct.

Reply:
Ok, cool.